lightly/api/download.py

```diff
@@ -110,24 +113,32 @@ def download_video_frame(
     _check_av_available()
     if timestamp < 0:
         raise ValueError(f"Negative timestamp is not allowed: {timestamp}")
+    if time_unit not in ('sec', 'pts'):
+        raise ValueError(f"time_unit must be 'sec' or 'pts' but is {time_unit}")
 
     container = av.open(url)
     stream = container.streams.video[video_channel]
     stream.thread_type = thread_type
-    offset = int(timestamp / stream.time_base)
+    
+    if time_unit == 'sec':
+        offset = int(timestamp / stream.time_base) + 1
```

Comment:
Could you elaborate on the `+ 1` part? Is this required to make the offsets compatible?